.github/ISSUE_TEMPLATE/new-device-support.md

```diff
@@ -24,6 +24,11 @@ Please post the output of
     ls -lah /dev/input/by-id/
 ```
 
+```
+Please post the output of
+    usbhid-dump -m 1532 -ed
+```
```

Comment:
Given the output is quite lengthy, maybe makes to sense to wrap in `<details></details>` tags so it's collapsed by default?

Reply:
sure, good idea.